cmd/eval.go

```diff
@@ -219,6 +220,7 @@ Set the output format with the --format flag.
 	evalCommand.Flags().BoolVarP(&params.partial, "partial", "p", false, "perform partial evaluation")
 	evalCommand.Flags().StringArrayVarP(&params.unknowns, "unknowns", "u", []string{"input"}, "set paths to treat as unknown during partial evaluation")
 	evalCommand.Flags().StringArrayVarP(&params.disableInlining, "disable-inlining", "", []string{}, "set paths of documents to exclude from inlining")
+	evalCommand.Flags().BoolVarP(&params.shallowInlining, "shallow-inlining", "", false, "disable inlining of rules that depend on unknowns")
```

Comment:
💭  I wonder if it would make sense at some point to support the same optimization level options in `eval` as with `build`. Its nice to have the flexible knobs and levers on `eval` to turn on/off stuff but for someone trying to reproduce something with a bundle at level 1 vs 2 it maybe isn't obvious what combination of flags it needs to do that.

Reply:
Adding support for the optimization level flag to eval would be nice. I always think of opa eval as the CLI for topdown so usually it just gets a copy of the params supported by that package. There's no reason we couldn't also suppose the optimization levels though.